cpp/core/CMakeLists.txt

```diff
@@ -141,7 +141,13 @@ if(USE_AVX512)
 endif()
 
 # Set up Proto
-set(PROTO_OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/proto")
+if (NOT DEFINED VELOX_HOME)
+  set(VELOX_HOME ${CMAKE_SOURCE_DIR}/../ep/build-velox/build/velox_ep)
+  message(STATUS "Set VELOX_HOME to ${VELOX_HOME}")
+endif()
+
+set(PROTO_OUTPUT_DIR "${VELOX_HOME}/velox/substrait/proto/")
```

Comment:
Hmm, it's more suitable to maintain substrait proto files under Gluten since CH backend will use it also, instead of Velox ones.

One option is to maintain exact same proto files under OAP Velox with Gluten ones. And add one Github actions to verify proto contents or manually check during code reviews.

Reply:
Thanks, I will create a new PR to use substrait proto files under Gluten. At present, this method will encounter some problems. Because the proto file created by substrait has different MACRO. we will met `redefinition problem`. This will take a while to research.